x-pack/plugins/apm/public/components/shared/dependencies_table/index.tsx

```diff
@@ -58,7 +58,7 @@ export function DependenciesTable(props: Props) {
     nameColumnTitle,
     status,
     compact = true,
-    hidePerPageOptions = true,
+    hidePerPageOptions = false,
```

Comment:
Since you set `hidePerPageOptions:false` as default value in `ManagedTable ` do you need to pass this prop as here?

Reply:
DependenciesTable and TransactionsTable are both used on the service overview and there we don't want to show the page options, so passing the prop form there is needed and that's why I set a default also on those components.